backend/pipelines/bmd_scraper/silver/transform.py

```diff
@@ -0,0 +1,411 @@
+            # Remove special characters
+            clean_col = ''.join(c if c.isalnum() or c == '_' else '_' for c in clean_col)
+            # Remove consecutive underscores
+            while '__' in clean_col:
+                clean_col = clean_col.replace('__', '_')
+            # Remove leading/trailing underscores
+            clean_col = clean_col.strip('_')
+            
+            column_mapping[col] = clean_col
+        
+        # Rename columns
+        df = df.rename(columns=column_mapping)
+        
+        # Log the column mapping
+        logger.info(f"Column mapping: {column_mapping}")
+        
+        return df
+    
+    def clean_data(self, df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Clean and normalize the data.
+        
+        Args:
+            df: DataFrame with raw data
+            
+        Returns:
+            Cleaned DataFrame
+        """
+        logger.info("Cleaning and normalizing data")
+        
+        # Make a copy to avoid modifying the original
+        df_clean = df.copy()
+        
+        # Clean text columns (strip whitespace)
+        for col in df_clean.select_dtypes(include=['object']).columns:
+            df_clean[col] = df_clean[col].astype(str).str.strip()
+        
+        # Normalize status fields if they exist
+        status_columns = [col for col in df_clean.columns if 'status' in col.lower() 
+                         or 'godkendelse' in col.lower() or 'tilladelse' in col.lower()]
+        
+        for col in status_columns:
+            df_clean[col] = df_clean[col].map(self.status_mapping).fillna(df_clean[col])
+        
+        # Handle duplicate rows
+        if df_clean.duplicated().any():
+            logger.warning(f"Found {df_clean.duplicated().sum()} duplicate rows, removing them")
+            df_clean = df_clean.drop_duplicates()
+        
+        return df_clean
+    
+    def parse_dates(self, df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Parse and standardize date columns.
+        
+        Args:
+            df: DataFrame with raw data
+            
+        Returns:
+            DataFrame with standardized date columns
+        """
+        logger.info("Parsing date columns")
+        
+        # Identify potential date columns
+        date_columns = [col for col in df.columns 
+                       if any(date_term in col.lower() for date_term in 
+                             ['date', 'dato', 'godkendelsesdato', 'udløbsdato', 'expiry'])]
+        
+        for col in date_columns:
+            try:
+                # Try converting to datetime
+                df[col] = pd.to_datetime(df[col], errors='coerce')
+                logger.info(f"Converted {col} to date format")
+            except Exception as e:
+                logger.warning(f"Could not convert {col} to date: {e}")
+        
+        return df
+    
+    def apply_types(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
the numbers with decimals were returned in Danish format (with comma instead of colon). Is that intended behavior? I think it's best to return it in US format.